README.md

```diff
@@ -2,6 +2,33 @@ XRP On Ledger Settlement engine as per the proposed [Settlement RFC](https://git
 
 Note the RFC has yet to be merged and thus the implementation could change to meet the spec at a later date. 
 
+## Build the Engine
+To build this engine, issue the following command:
+
+```bash
+npm install && ./node_modules/typescript/bin/tsc && npm link
```

Comment:
This can just be 
`bash
npm install && npm run build && npm link
`

Reply:
Fixed!